src/Common/src/TypeSystem/Common/Utilities/LockFreeReaderHashtable.cs

```diff
@@ -36,11 +36,11 @@ abstract public class LockFreeReaderHashtable<TKey, TValue> where TValue : class
         private static TValue[] s_hashtableInitialArray = new TValue[1];
 
         /// <summary>
-        /// _count represents the current count of elements in the hashtable
-        /// _count is used in combination with _resizeCount to control when the 
+        /// Count represents the current count of elements in the hashtable
```

Comment:
Well, the _resizeCount comment is a bit unfortunate from a private property, but as this property declaration is both the variable and public accessor, I don't see a good way around it. I think its good enough for now.